### PR TITLE
Modify C-RTK2 hp error message and add to droncan peripherals

### PR DESCRIPTION
**Pull request by cuhome**
image
Modify display error and add C-RTK2 HP to droncan peripheral.

### CODE REVIEW

File: common/source/docs/common-uavcan-peripherals.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ DroneCAN was created to continue the development of the widely used UAVCAN v0 pr
     CUAV Neo v2 Pro DroneCAN GPS <common-cuav-neo-v2-pro>
     CUAV Neo 3 Pro DroneCAN GPS <common-cuav-neo-3-pro>
     CUAV C-RTK2 PPK and RTK receiver<common-cuav-c-rtk2>
+    CUAV C-RTK2 HP Haeding and RTK receiver<common-cuav-c-rtk2-hp>
```
Comment (Hwurzburg) [inline common/source/docs/common-uavcan-peripherals.rst:20]:
```suggestion
    CUAV C-RTK2 HP Heading and RTK receiver<common-cuav-c-rtk2-hp>
```

**Comment by Hwurzburg** [comment]
@cuhome can we squash the commits to one...thanks

just FYI, I missed in my review that this page includes that competitive comparison chart....even though the Ublox data is publicly available, I believe ArduPilot does not put competitive comparison charts, selector guides, etc. or other marketing oriented information in the docs...ie we want to avoid showing preference....your side of the chart is fine, and , of course, you can put any marketing oriented info in your own website docs

since the data is objective and not subjective, I have put the issue before the team for a decision

**Comment by Hwurzburg** [comment]
Yes, we would like to remove the F9P side of the table...for the reasons stated above...I can do it and squash, if you like....thanks

**Comment by cuhome** [comment]
@Hwurzburg I agree with your edit, please merge it.thanks for your help

**Pull request merged**